test/e2e_node/jenkins/benchmark/benchmark-config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+images:
+  containervm-density1:
+    image: e2e-node-containervm-v20160321-image
```

Comment:
Ignore above comment. We did re-create those images after running https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/environment/setup_host.sh and save it for e2e testings.